### ISSUE TITLE
Add settings option which languages to include/exclude

### ISSUE THREAD

**Issue by bzrncev** [imliam/vscode-inline-parameters#15]
Will be good to have option to exclude some of the languages this extension offers. For example, me personally want to show parameter hints only in PHP, and not in JS. 

What do you think?

**Comment by imliam** [comment]
Makes sense to me - I'll look into adding per-language configuration so you can configure it separately for each language.
